cccs-build/superset/Dockerfile

```
@@ -1,8 +1,13 @@
+# Vault CA container import
+ARG VAULT_CA_CONTAINER=uchimera.azurecr.io/cccs/hogwarts/vault-ca:master_2921_22315d60
+FROM $VAULT_CA_CONTAINER AS vault_ca
+
 FROM uchimera.azurecr.io/cccs/superset-base:cccs-1.4_20220411144824_b3288
 
 USER root
 
 COPY requirements.txt /tmp
+COPY analytical-platform-requirements.txt /tmp
```

We should combine these two lines to avoid creating an extra layer here
```suggestion
COPY *requirements.txt /tmp/
```